Sources/Tentacle/Client.swift

```diff
@@ -26,8 +27,8 @@ extension URL {
     
     internal init<Value>(_ server: Server, _ request: Request<Value>, page: UInt? = nil, perPage: UInt? = nil) {
         let queryItems = [ ("page", page), ("per_page", perPage) ]
-            .filter { _, value in value != nil }
-            .map { name, value in URLQueryItem(name: name, value: "\(value!)") }
+            .filter { (_, value) -> Bool in value != nil }
+            .map { (name, value) -> URLQueryItem in URLQueryItem(name: name, value: "\(value!)") }
```

Comment:
You should be able to remove these return types?

Reply:
Done!